todoapp/build.gradle

```diff
@@ -24,12 +24,12 @@ task clean(type: Delete) {
 ext {
     // Sdk and tools
     minSdkVersion = 10
-    targetSdkVersion = 24
-    compileSdkVersion = 24
-    buildToolsVersion = '24.0.2'
+    targetSdkVersion = 25
```

Comment:
shouldn't we do this as part of a different PR? same with the Support Library Versions